privacy-snapshot.json

```diff
@@ -23,18 +24,22 @@
   "gas.api.cx.metamask.io",
   "github.com",
   "goerli.infura.io",
+  "lattice.gridplus.io",
+  "localhost:12345",
```

Comment:
Note for other reviewers, these two localhost entries appear to be the only additions. The other changes are sorting and removing duplicates.